pkg/roachprod/install/cockroach.go

```diff
@@ -1671,3 +1685,10 @@ func getEnvVars() []string {
 func SuppressMetamorphicConstantsEnvVar() string {
 	return config.DisableMetamorphicTestingEnvVar
 }
+
+// ClusterHook exposes a way to run functions in between steps normally
+// orchestrated by the framework.
```

Comment:
In this context, the only "framework" is `roachprod`; maybe paraphrase to say that it's merely a hook that's executed locally before cockroach process is started in the cluster.

Reply:
Done.